test/functional/provisioning_templates_controller_test.rb

```diff
@@ -103,7 +103,7 @@ class ProvisioningTemplatesControllerTest < ActionController::TestCase
     Setting[:unattended_url] = "http://foreman.unattended.url"
     @request.env['HTTP_REFERER'] = provisioning_templates_path
 
-    ProxyAPI::TFTP.any_instance.expects(:create_default).with(has_entry(:menu, regexp_matches(/ks=http:\/\/foreman.unattended.url:80\/unattended\/template/))).returns(true)
+    ProxyAPI::TFTP.any_instance.expects(:create_default).with(has_entry(:menu, regexp_matches(/ks=http:\/\/foreman.unattended.url\/unattended\/template/))).returns(true)
```

Comment:
Is this still working with non-standard ports?  i.e. is it hiding :80 automatically, or is it broken for all ports?